modules/appeals_api/app/controllers/appeals_api/v1/decision_reviews/notice_of_disagreements/evidence_submissions_controller.rb

```diff
@@ -5,9 +5,17 @@ module DecisionReviews
     module NoticeOfDisagreements
       class EvidenceSubmissionsController < AppealsApi::ApplicationController
         skip_before_action :authenticate
+        before_action :nod_id_present?, only: :create
```

Comment:
I'm not really a fan of the reliance on callbacks here to manage flow of the controller logic. In my opinion, these two guard clauses should exist within the controller action.